packages/monaco/src/browser/monaco-workspace.ts

```diff
@@ -288,7 +297,7 @@ export class MonacoWorkspace {
                 const uri = monaco.Uri.parse(key);
                 let eol: EndOfLineSequence | undefined;
                 const editOperations: monaco.editor.IIdentifiedSingleEditOperation[] = [];
-                const minimalEdits = await StandaloneServices.get(IEditorWorkerService).computeMoreMinimalEdits(uri, value.map(v => v.textEdit));
+                const minimalEdits = await StandaloneServices.get(IEditorWorkerService).computeMoreMinimalEdits(uri, value.map(this.transformSnippetStringToInsertText));
```

Comment:
Suggestion: Since the `transformSnippetStringToInsertText` function isn't bound to anything, references to `this` in there will fail. To prevent such an error in the future, either use a lambda or transform `transformSnippetStringToInsertText` to a field:

```ts
private transformSnippetStringToInsertText = (resourceEdit: MonacoResourceTextEdit): TextEdit & { insertAsSnippet?: boolean } => {
    ...
}  
```